change/office-ui-fabric-react-2019-07-16-17-27-21-v-mare-Checkbox-indeterminate-state.json

```diff
@@ -0,0 +1,8 @@
+{
+  "comment": "Updated API documentation",
```

Comment:
Change the comment in with the one from the change file that needs to be removed.